### ISSUE TITLE
Percent signs are stripped from titles

### ISSUE THREAD

**Issue by armandn** [justintadlock/mythic#78]
If a title contains a percent sign, e.g. "Nasdaq up 25%", the "%" is stripped from the title (only in posts archives, single titles are OK).

It looks like the problem is in  functions-post.php, in render_permalink(): there's this code:
```
	$html = sprintf(
		'<a class="%s" href="%s">%s</a>',
		esc_attr( $args['class'] ),
		esc_url( $url ),
		sprintf( $args['text'], esc_url( $url ) )
	);
```
I don't understand the purpose of `sprintf( $args['text'], esc_url( $url ) )`, but replacing it with just `$args['text']` seems to fix the problem.

**Comment by justintadlock** [comment]
Are you posting in the right repository?

Is this something in Hybrid Core? https://github.com/justintadlock/hybrid-core

**Comment by armandn** [comment]
Yes, sorry, it's Hybrid Core. I forgot that under the hood Mythic uses Hybrid Core that has a separate repo.

**Comment by armandn** [comment]
Do you want me to post the issue on Hybrid Core and close this one?
